test/queries/test_asset.py

```diff
@@ -0,0 +1,84 @@
+"""
+    Tests assets query.
+"""
+import os
+from kili.client import Kili
+
+api_key = os.getenv("KILI_USER_API_KEY")
+api_endpoint = os.getenv("KILI_API_ENDPOINT")
+kili = Kili(api_key=api_key, api_endpoint=api_endpoint)
+
+TEST_CASES = [
+    {
+        "case": "AAU, When I query assets with first=50, I get the first 50 assets",
```

Comment:
I feel like a test is missing, when fetching as a generator without specifying first, to test if there aren't any problems fetching all assets